src/main/java/mate/jdbc/dao/ManufacturedDaoImpl.java

```diff
@@ -0,0 +1,113 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.connection.DbConnector;
+import mate.jdbc.exception.DataException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+
+@Dao
+public class ManufacturedDaoImpl implements ManufacturedDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertManufacturerRequest =
+                "INSERT INTO manufacturers(name, country) values (?, ?);";
```

Comment:
All SQL keywords should be uppercase. Don't forget about spaces 
![image](https://user-images.githubusercontent.com/32339836/227153795-74734644-926d-4c8a-87cb-fc7dab8e18ba.png)
```suggestion
                "INSERT INTO manufacturers (name, country) VALUES (?, ?);";
```